app/configurator/components/field.tsx

```diff
@@ -2,7 +2,7 @@ import { t } from "@lingui/macro";
 import { I18n } from "@lingui/react";
 import { Box, Flex } from "@theme-ui/components";
 import get from "lodash/get";
-import { ChangeEvent, ReactNode, useCallback } from "react";
+import React, { ChangeEvent, ReactNode, useCallback } from "react";
```

Comment:
Please don't use the default React export anymore, it will be removed in the future.